src/bin/driver.rs

```diff
@@ -227,7 +229,7 @@ mod app {
         cpu_temp_sensor: stabilizer::hardware::cpu_temp_sensor::CpuTempSensor,
         header_adc_conversion_scheduled: TimerInstantU64<MONOTONIC_FREQUENCY>, // auxillary local variable for exact scheduling
         driver_dac: [driver::dac::Dac<driver::Spi1Proxy>; 2],
-        lm75: lm75::Lm75<I2c1Proxy, lm75::ic::Lm75>,
+        lm75: [lm75::Lm75<I2c1Proxy, lm75::ic::Lm75>; 2],
```

Comment:
```suggestion
        driver_temp: lm75::Lm75<I2c1Proxy, lm75::ic::Lm75>,
        header_temp: lm75::Lm75<I2c1Proxy, lm75::ic::Lm75>,
```

Otherwise fine.

Reply:
I had that but thought you'd prefer to save the extra line ;)

Reply:
But in the resources struct I'll leave it as an array?